app/services/create_cluster_stats_service.rb

```diff
@@ -36,6 +36,25 @@ def call
 
   private
 
+  def epochs_annual_roi
+    if last_epochs.count > 0
+      epoch_duration = (last_epochs.first.created_at - last_epochs.second.created_at)
+      return 0 unless epoch_duration.to_f > 0
+      epoch_average_roi = last_epochs.map do |epoch|
+        if epoch.total_rewards && epoch.total_active_stake
+          (epoch.total_rewards / epoch.total_active_stake.to_f) * 100
+        end
+      end.compact.average
+
+      epochs_annual_roi = (1.year.to_i / epoch_duration.to_f) * epoch_average_roi
```

Comment:
```suggestion
      epochs_annual_roi = (1.year.to_i / epoch_duration.to_f) * epoch_average_roi.to_f
```

Reply:
This still need to be fixed. After deploying to production initially this value will be nil and will throw errors.